### DISCUSSION TITLE
✅ 🚀 Feature Request: Add Filterbar option for loop grid Widget like as portfolio Widget

### DISCUSSION THREAD

**Discussion by stratos-code** [elementor/elementor#22648] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Actually, We can easly add a filterable table with Portfolio Widget, but we can't easly customize the card style and work with custom fields. And we can use custom card with templates but can't add a filter without a plugin, and also, it's hard to integrate that functionality.

### Describe the solution you'd like

Like the title says... Add a Filterbar option for loop grid Widget like as portfolio Widget. That we can select the filter by diferents taxonomies... even custom taxonomies.

### Additional context

![image](https://github.com/elementor/elementor/assets/53635026/6cb4dae6-f6f9-462b-91d8-67642f3a2b25)

**Comment by 79ho3ein** [comment]
This is on the [Elementor roadmap](https://elementor.com/roadmap/). We have to wait for that.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.15.0 with the new experimental Taxonomy Filter widget! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
